CSharp/MineStat/MineStat.cs

```diff
@@ -30,6 +30,15 @@
 using System.Xml;
 using System.Xml.Linq;
 using System.Xml.XPath;
+using System.Threading.Tasks;
+using System.Net;
+
+#if NETSTANDARD2_0
+using DnsClient;
+#elif NET46
+using ARSoft.Tools.Net;
+using ARSoft.Tools.Net.Dns;
+#endif
```

Comment:
Would we want to handle all other cases with an `#else` here?

Reply:
I don't see how `#else` can be used logically here. The only thing that comes to mind is a construction like this:

```
#else
#error This target framework is not supported by this code. Please ensure the target is .NET Standard 2.0 or greater, or .NET Framework 4.6 or greater.
```